flutter-studio/src/io/flutter/project/FlutterSettingsStep.form

```diff
@@ -112,6 +112,30 @@
           <text value="Applications and plugins need to generate platform-specific code"/>
         </properties>
       </component>
+      <component id="5bff0" class="com.intellij.ui.components.JBLabel" binding="myAndroidXLabel">
+        <constraints>
+          <grid row="9" column="0" row-span="1" col-span="1" vsize-policy="0" hsize-policy="0" anchor="8" fill="0" indent="0" use-parent-layout="false"/>
+        </constraints>
+        <properties>
+          <font style="1"/>
+          <text value="AndroidX"/>
+        </properties>
+      </component>
+      <component id="f10cb" class="javax.swing.JCheckBox" binding="myUseAndroidxCheckBox">
+        <constraints>
+          <grid row="10" column="0" row-span="1" col-span="1" vsize-policy="0" hsize-policy="3" anchor="8" fill="0" indent="0" use-parent-layout="false"/>
+        </constraints>
+        <properties>
+          <text value="Use androidx.* artifacts"/>
```

Comment:
nit: maybe provide a tooltip explaining why this may be a good idea.

Reply:
Sounds like a good idea. Neither IJ or AS has such a tooltip so I'm not sure exactly what to say. "This is going to be the default so you might as well use it" ? That doesn't sound quite right.